tests/data/sample_poses.py

```diff
@@ -0,0 +1,86 @@
+"""Sample poses for testing the averaging algorithms.
+
+Authors: Ayush Baid
+"""
+import copy
+from typing import Dict, List, Tuple
+
+import numpy as np
+from gtsam import Pose3, Rot3
+
+
+def generate_relative_from_global(
+    wTi_list: List[Pose3], pair_indices: List[Tuple[int, int]]
+) -> Dict[Tuple[int, int], Pose3]:
+    """Generate relative poses from global poses.
+
+    Args:
+        wTi_list: global poses.
+        pair_indices: pairs (i1, i2) to construct relative poses for.
+
+    Returns:
+        Dictionary (i1, i2) -> i2Ti1 for all requested pairs.
+    """
+    return {(i1, i2): wTi_list[i2].between(wTi_list[i1]) for i1, i2 in pair_indices}
+
+
+"""4 poses in the circle of radius 5m, all looking at the center of the circle.
+
+For relative poses, each pose has just 2 edges, connecting to the immediate neighbors.
+"""
+CIRCLE_TWO_EDGES_GLOBAL_POSES = [
+    Pose3(Rot3.RzRyRx(0, 0, 0), np.array([0, 0, 0])),
+    Pose3(Rot3.RzRyRx(np.deg2rad(90), 0, 0), np.array([0, 5, 5])),
+    Pose3(Rot3.RzRyRx(np.deg2rad(180), 0, 0), np.array([0, 0, 10])),
+    Pose3(Rot3.RzRyRx(np.deg2rad(270), 0, 0), np.array([0, -5, 5])),
+]
+
+CIRCLE_TWO_EDGES_RELATIVE_POSES = generate_relative_from_global(
+    CIRCLE_TWO_EDGES_GLOBAL_POSES, [(1, 0), (2, 1), (3, 2), (0, 3)]
+)
+
+"""4 poses in the circle of radius 5m, all looking at the center of the circle.
+
+For relative poses, each pose is connected to every other (3) pose.
+"""
+CIRCLE_ALL_EDGES_GLOBAL_POSES = copy.copy(CIRCLE_TWO_EDGES_GLOBAL_POSES)
+
+CIRCLE_ALL_EDGES_RELATIVE_POSES = generate_relative_from_global(
+    CIRCLE_TWO_EDGES_GLOBAL_POSES, [(0, 1), (0, 2), (0, 3), (1, 2), (1, 3), (2, 3)]
+)
+
+"""3 poses on a line, simulating forward motion, with large translations and no relative rotation.
+
+For relative poses, we have a fully connected graph.
+"""
+LINE_LARGE_EDGES_GLOBAL_POSES = [
+    Pose3(Rot3(), np.array([0, 0, 0])),
```

Comment:
My suggestion was to use a non-zero constant rotation here, so something like Rot3.RzRyRx(rx, ry, rz), where rx, ry and rz are constants for all three poses. Also, might be better to have the translations start at a different origin, something like camera1 = [1,1,1], camera2 = [2,2,2], camera3 = [3,3,3]. 
I understand we lose simplicity if we do this. 
But we can check whether we are able to align GT poses to estimated poses using our alignment methods. Because in the estimated pose, camera1 will be at [0, 0, 0]. 
But if you want to keep it simple and do not want to test alignment, I am fine with this.

Reply:
I did change this to have non-zero defaults but forgot to push the code. Is it better now?

I have some non-zero values, but not all coordinates are at a non-zero value.

Reply:
Yeah, looks good now, although a higher value might be better for default translation. Maybe 2 instead of 0.2, as estimates themselves could be off by 0.2.

Reply:
Cool I made them 2.